tests/album-read.test.js

```diff
@@ -0,0 +1,75 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('read album', () => {
+  let db;
+  let albums;
+
+  beforeEach(async () => {
+    db = await getDb();
+
+    const [{ insertId }] = await db.query('INSERT INTO Artist (name, genre) VALUES(?, ?)', [
+        'Taylor Swift',
+        'Country',
+      ])
```

Comment:
For longer lines like this it can be made easier to read by separating each statement out onto its own line:
```
    const [{ insertId }] = await db.query(
      'INSERT INTO Artist (name, genre) VALUES(?, ?)',
      ['Taylor Swift', 'Country']
    );
```

(incidentally the formatter. will do this for you)